src/hotspot/cpu/riscv/gc/shenandoah/c1/shenandoahBarrierSetC1_riscv.cpp

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2018, 2019, Red Hat, Inc. All rights reserved.
  * Copyright (c) 2020, 2021, Huawei Technologies Co., Ltd. All rights reserved.
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
This should be backed out, since it seems that there is no (other) change to this fie.

Reply:
Yes. And also from files which were changed by non-Amazon employees only, please.

Reply:
Thanks, MArtin. Yes, we have noted that there were a few other files that were inadvertently caught in a opyright header dragnet. These will be reviewed and fixed in https://bugs.openjdk.org/browse/JDK-8309392 .

Reply:
I'm fixing this copyright notice and others.